src/utils/parseTxData/api.ts

```diff
@@ -21,7 +21,9 @@ export const api = async (address: string) => {
     res[jettonInfo.symbol] = {
       ...jettonInfo,
       pnlPercentage,
-      chart: (slicedChart.length >= 2 ? slicedChart : chart).reverse().map(entity => [entity[0], normalizePrice(entity[1], jettonInfo.decimals)]),
+      chart: (slicedChart.length >= 2 ? slicedChart : chart)
+        .reverse()
+        .map(entity => [entity[0], normalizePrice(entity[1], jettonInfo.decimals)]),
```

Comment:
This logic should not be here. return the literal price value, and normalize it on the frontend. Less load on the server + eventually I want to turn it into a string (not valid number at all), e.g. something like `0.₉1234`

Reply:
Actually, frontend is completely ok with displaying a string. But I guess our clients have decent hardware and don't care about some client-side transformations anyway.